devicetypes/vision-nameseca/zp3111-4-in-1-motion-sensor.src/zp3111-4-in-1-motion-sensor.groovy

```diff
@@ -0,0 +1,237 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "ZP3111 4-in-1 Motion sensor", namespace: "vision_nameseca", author: "SmartThings", mnmn: "SmartThings", vid: "generic-motion-8", ocfDeviceType: "x.com.st.d.sensor.motion") {
```

Comment:
change name and namespace.
name: "Zooz 4-in-1 sensor", namespace: "smartthings" 
->name: "ZP3111 4-in-1 Motion sensor", namespace: "vision_nameseca"